src/components/notification/Notification.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Button, notification } from 'antd'
+import { NotificationPlacement } from 'antd/es/notification/interface'
+
+type Props = Expand<{
+  type: 'success' | 'info' | 'warning' | 'error'
+  message: string
+  description: string
+  placement: NotificationPlacement
+  children?: React.ReactNode
+}>
+
+const openNotification = (props: Props) => {
+  const { type, message, description, placement, children } = props
+  notification[type]({
+    message,
+    description,
+    placement,
+    btn: children || null,
+  })
+}
+
+const Notification = ({
+  type,
+  message,
+  description,
+  placement,
+  children,
+}: Props) => {
+  return (
+    <Button
+      onClick={() =>
+        openNotification({
+          type,
+          message,
+          description,
+          placement,
+          children,
+        })
+      }
+    >
+      Show Notification
+    </Button>
```

Comment:
Is `Notification` supposed to be more of a code sample than a component we use? If not, I don't think this button is supposed to be a part of the component.